src/main/java/com/tagtraum/perf/gcviewer/view/ModelDetailsPanel.java

```diff
@@ -109,6 +118,13 @@ public void setModel(GCModel model) {
         }
         concurrentGcEventModel.setModel(model.getConcurrentEventPauses(), totalPause, false);
 
+        if (model.size() > 1 && model.getGcEventCauses().size() == 0) {
+            remove(gcCausesTable);
+        }
+        else {
+        	gcCausesModel.setModel(model.getGcEventCauses(), totalPause, false);
```

Comment:
Since, currently, there is no pause time in the causes, this line might cause the weird numbers. But first, let's clarify on the right model (see my comment in the issue).